src/Core/Components/DateTime/FluentDatePicker.razor

```diff
@@ -1,6 +1,16 @@
-@namespace Microsoft.FluentUI.AspNetCore.Components
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
 @inherits FluentCalendarBase
 
+@code {
```

Comment:
I think using `yyyyy` is a good solution. But could you move this method to the .cs class.
Except for the demo site, where we prefer clear explanations, in the Core project we want to put the C# code in the Behind (.cs) part and the HTML tags in the razor file.